src/Analyzers/VisualBasic/CodeFixes/VisualBasicCodeFixes.projitems

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)AddAccessibilityModifiers\VisualBasicAddAccessibilityModifiersCodeFixProvider.vb" />
     <Compile Include="$(MSBuildThisFileDirectory)ConvertAnonymousTypeToTuple\VisualBasicConvertAnonymousTypeToTupleCodeFixProvider.vb" />
+    <Compile Include="$(MSBuildThisFileDirectory)ConvertTypeOfToNameOf\VisualBasicConvertTypeOfToNameOfCodeFixProvider.vb" />
```

Comment:
VB side should likely be called ...ConvertGetTypeToNameOf.  But i'm not going to push hard on that if you prefer this.